assignment/Module6/ClassDeleteTrigger.apxt

```diff
@@ -0,0 +1,14 @@
+trigger ClassDeleteTrigger on Class__c (before delete) {
+    List<Class__c> classes = Trigger.Old;
+    List<Id> classIds = new List<String>();
+    for(Class__c cls : classes){
+        classIds.add(cls.Id);
+    }
+	List<AggregateResult> listCount = new List<AggregateResult>([select count(Id) recordCount, class__c classId from student__c where class__c in :classIds and sex__c = 'Female' group by class__c ]);
+    for(AggregateResult rs : listCount){
+        if(((Integer)rs.get('recordCount'))>1){
+            Class__c clazz = classes.get(0);
```

Comment:
What if multiple classes are being deleted at same time. would `get(0)` work in that case also?